src/core/services/NotificationEmitterApi.ts

```diff
@@ -15,12 +15,32 @@ export const remove: Remove = (id) => emitter.emit('remove_notification', { id }
 export const modify: Modify = (id: string, { params, config }) =>
   emitter.emit('modify_notification', { id, params, config })
 
-export const notify: Notify = (notificationType, setup) => {
+export const notify: Notify = (
+  notificationType: any,
+  titleOrSetup: any,
+  descriptionOrSetup?: any,
+  setup?: any
```

Comment:
Is there a bettor option than 'any' here? Could this be improved by adding more specific types here?